examples/modal.py

```diff
@@ -0,0 +1,67 @@
+import discord
+from discord import app_commands, Object, ui
+
+# Just default intents and a `discord.Client` instance
+# We don't need a `commands.Bot` instance because we are not
+# creating text-based commands.
+intents = discord.Intents().default()
+client = discord.Client(intents=intents)
+
+# We need an `discord.app_commands.CommandTree` instance to register slash commands
+tree = app_commands.CommandTree(client)
+
+# The guild in which this slash command will be registered.
+# As global commands can take up to an hour to propagate, it is ideal
+# to test it in a guild.
+test_guild = Object(ID_HERE)
+
+@client.event
+async def on_ready():
+    print(f"Connected as {client.user}")
+    # Sync the slash commands with Discord.
+    await tree.sync(guild=test_guild)
+
+class Feedback(ui.Modal, title='Feedback'):
+    # This will be a short input, where the user can enter their name
+    # It will also be required, denoted by the `required=True` kwarg
+    name = ui.TextInput(
+        label='Name', 
+        style=discord.TextStyle.short, 
+        placeholder='Your name here...',
+        required=True,
+        custom_id='modal_name'
+    )
```

Comment:
```suggestion
    name = ui.TextInput(label='Name', placeholder='Your name here...')
```

There's no need to re-specify defaults. Examples are meant to simple.